lxc/Deploy.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+# Initially need to config LXD on host machine. Run:
+# sudo lxd init
+# Init config should be like:
+#
+# config:
+#   images.auto_update_interval: "0"
+# networks: []
+# storage_pools:
+# - config:
+#     size: 30GiB
+#   description: ""
+#   name: default
+#   driver: zfs
+# profiles:
+# - config: {}
+#   description: ""
+#   devices:
+#     root:
+#       path: /
+#       pool: default
+#       type: disk
+#   name: default
+# projects: []
+# cluster: null
+
+# Possible errors:
+
+# 1) If you get keyring error like: The specified keyring /var/cache/lxc/debian/archive-key.gpg may be incorrect or out of date
+# Just download the new keys and import it with:
+#
+# wget "https://ftp-master.debian.org/keys/release-10.asc"
+# sudo gpg --no-default-keyring --keyring /var/cache/lxc/debian/archive-key.gpg --import release-10.asc
+
+# 2) If you get an error tryint to run virsh commands like:
+# error: failed to connect to the hypervisor
+# error: Failed to connect socket to '/var/run/libvirt/libvirt-sock': No such file or directory
+#
+# Install kvm with:
+# sudo apt install qemu qemu-kvm libvirt-clients libvirt-daemon-system virtinst bridge-utils
+# sudo systemctl enable libvirtd
+# sudo systemctl start libvirtd
+
+#export USER=abelyak
+
+export CONTAINERS=10
+export NETPREF=192.168.122
+
+
+if [[ ! $USER ]]; then
+   echo "Specify USER variable"
+   exit 1
+fi
+
+if [ `whoami` != 'root' ]; then
+    echo "Run as root"
+    exit 2
+fi
+
+
+
+function setup {
+	echo "01. Create"
+	for (( i=1; i<=$CONTAINERS; i++ )) do lxc-create -n n$i -t debian -- --release buster; done
```

Comment:
What about user's privileges for `lxc-command` and other commands? Does a user require `sudo` for them, or does it need to belong to some group? These commands are not available for everybody in the system.

Reply:
README.md: requires root privileges